src/modules/data/BitField.ts

```diff
@@ -14,23 +14,40 @@
 import { Validator, IBitField } from "./validator";
 
 /**
- * The class that defines and parses the BitField of a block.
+ * The class that defines the BitField of a block.
  * Convert JSON object to TypeScript's instance.
  * An exception occurs if the required property are not present.
  */
 export class BitField
 {
-    _storage: number[] = [];
+    /**
+     * The storage with bit data
+     */
+    public _storage: number[];
```

Comment:
Is there a reason to start with the under-bar in the public variable?
I can't understand the style convention.

Reply:
This is using `_storage` to keep it the same as the JSON format received from Agora.